composer.json

```diff
@@ -29,6 +29,7 @@
         "sonata-project/core-bundle": "~2.3,>=2.3.1"
     },
     "require-dev": {
+        "rhumsaa/uuid": "^2.8",
```

Comment:
I had to use the old uuid lib from Rhumsaa instead of Ramsey's to keep the tests compatible with php 5.3

I use the lib because I use a uuid class to generate uuid (and have it mapped by doctrine) to test the changes.